src/Tribe/Tickets.php

```diff
@@ -784,6 +806,8 @@ public static function get_all_event_tickets( $post_id ) {
 				$tickets = array_merge( $tickets, $obj->get_tickets( $post_id ) );
 			}
 
+			set_transient( $cache_key, $tickets, HOUR_IN_SECONDS );
```

Comment:
curious about the `HOUR_IN_SECONDS` choice here — if we're going to be clearing cache anytime the post is updated, maybe we can make this a longer time period? like six hours or something?